pkgs/development/web/newman/generate-dependencies.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p nodePackages.node2nix
+
+node2nix \
+  --node-env ../../node-packages/node-env.nix \
```

Comment:
I know many standalone node packages use this, but I think it's not needed. My gut feeling tells me it should be a different node-env for each such package.

Reply:
so just local to this folder ?

Reply:
yes.